main.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
+
+hamburger.addEventListener("click", () => {
+    hamburger.classList.toggle("active");
+    navMenu.classList.toggle("active");
+})
+
+document.querySelectorAll(".nav-link").forEach(link => link.addEventListener("click", () => {
+    hamburger.classList.remove("active");
+    navMenu.classList.remove("active");
+}))
```

Comment:
Please kindly check this code. When the Hamburger button (Mobile Nav bar) is clicked nothing is poping up please kindly fix this issue.